src/Stratis.SmartContracts.Tests.Common/MockChain/MockChainNode.cs

```diff
@@ -123,15 +123,30 @@ public HdAddress GetUnusedAddress()
         /// <summary>
         /// Send a normal transaction.
         /// </summary>
-        public Result<WalletSendTransactionModel> SendTransaction(Script scriptPubKey, Money amount)
+        public Result<WalletSendTransactionModel> SendTransaction(Script scriptPubKey, Money amount, int utxos = 0)
         {
+            Recipient[] recipients;
+
+            if (utxos > 0)
+            {
+                recipients = new Recipient[utxos];
+                for (int i = 0; i < recipients.Length; i++)
+                {
+                    recipients[i] = new Recipient {Amount = amount / utxos, ScriptPubKey = scriptPubKey};
```

Comment:
Should we add some checks here that amount / utxos is valid, eg. if amount is 1 sat and UTXOs is 3.